script.sh

```diff
@@ -13,6 +13,20 @@ echo '::group:: Installing tflint ... https://github.com/terraform-linters/tflin
 curl -sfL https://raw.githubusercontent.com/terraform-linters/tflint/master/install_linux.sh | TFLINT_VERSION="${INPUT_TFLINT_VERSION}" bash
 echo '::endgroup::'
 
+export TFLINT_PLUGIN_DIR="${TEMP_PATH}/.tflint.d/plugins"
+for RULESET in ${INPUT_TFLINT_RULESETS}; do
+  PLUGIN="tflint-ruleset-${RULESET}"
+  REPOSITORY="https://github.com/terraform-linters/${PLUGIN}"
+
+  echo "::group:: Installing tflint plugin for ${RULESET} ... ${REPOSITORY}"
+  mkdir -p "${TFLINT_PLUGIN_DIR}"
+  curl -sfL "${REPOSITORY}"/releases/latest/download/"${PLUGIN}"_"$(uname | sed 's/Darwin/darwin/; s/Linux/linux/')"_"$(uname -m | sed 's/x86_/amd/; s/aarch64/arm/')".zip -o "${PLUGIN}".zip && unzip "${PLUGIN}".zip -d "${TFLINT_PLUGIN_DIR}" && rm "${PLUGIN}".zip
```

Comment:
you might do:
```
uname | tr A-Z a-z
```
instead of using sed s///

Reply:
That could work! My thinking was that I am using `sed` to change the architecture string too, and I'd keep it consistent: people would see the first `sed` and, if they're not already familiar, they would understand it does a replace 🙂

Reply:
The current code wouldn't work on Windows :-)

Reply:
Hm... does the `tflint` installer work on Windows? It's named `intall_linux.sh` and has hardcoded non-Windows paths (`/usr`, `/tmp`, `/bin`), but it does have an option for Windows.

Reply:
🤷 

The reason I suggest things like `tr` is to avoid hard-coding a list. The fewer specific quirks the less work going forward in case the underlying system supports more things later.

It sounds like GitHub Runners will eventually be using msys2: https://github.com/msys2/setup-msys2#context which I think would include `/usr`, `/tmp`, and `/bin`, so I wouldn't just offhand assume that such things shouldn't work.

Reply:
I tested the script and I can confirm it works on Windows! I'll review and implement the changes needed to get this also running on Windows